database/seeders/OrderSeeder.php

```diff
@@ -13,10 +17,34 @@ class OrderSeeder extends Seeder
      */
     public function run(): void
     {
-        Order::create([
+        $order = Order::create([
             'order_date' => '2024-02-25',
+            'email'=> 'admin@mail.com',
             'lid_name' => 'jantje',
+            'order_status_id' => 4,
+            'user_id' => 13,
             'group_id' => 1
         ]);
+
+        $order->orderStatus = OrderStatus::first();
+
+        $orders[] = Order::factory()
+            ->has(OrderLine::factory())
+            ->has(DeliveryState::factory())
+            ->count(20)->create();
+        
+            array_push($orders, Order::factory()
+            ->has(OrderLine::factory())
+            ->has(DeliveryState::factory())
+            ->count(1)->create(['user_id' => 14, 'order_status_id' => Order::where('id', 1)->first()->id]));
```

Comment:
Stel er is geen user met id 14 omdat andere seed data verandert, dan crasht deze code. Waarom heb je dit hier hardcoded in staan?